src/test/java/plugins/GradlePluginTest.java

```diff
@@ -54,6 +57,8 @@
 
     /**
      * Verify the execution of a basic gradle build script.
+     *
+     * Prevent to run gradle daemon because attempt to clean processes after build put daemon into defunct mode and build timeout.
```

Comment:
The cause of strange behavior is gradle daemon process. It stayed in defunct mode after process killer tried to clean processes in the end of a build. I was not able to find what is the cause of this behavior of gradle daemon (or some reported bug which describes exactly the same situation). Since it does not happen in my environment, I think that it can be connected with docker in combination with specific environment of its host. 
I did not want to spend hours by searching because there is no need to run gradle daemon for ATH testing.
So I set --no-daemon switch for tests. 
@olivergondza are you alright with documentation for understanding why gradle daemon is not used?